components/ProgressView/src/MDCProgressView.m

```diff
@@ -282,4 +339,30 @@ - (void)updateTrackView {
   self.trackView.frame = self.hidden ? CGRectMake(0.0, size.height, size.width, 0.0) : self.bounds;
 }
 
+- (void)startAnimatingBar {
+  if (!_isIndeterminate) {
+    return;
+  }
+
+  // Cancel any other animations on the progress view.
+  [self.progressView.layer removeAllAnimations];
```

Comment:
Maybe checking if the progress view `isAnimating` or not and avoid remove/re-animating again if it is already animating?